index.html

```diff
@@ -88,73 +99,13 @@ <h1>Expand your career opportunities with Python</h1>
-              </h1>
-              <p>Avinash Jain, The Codex</p>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star"></span>
-              <span class="fa fa-star"></span>
-              <p>$94.99</p>
-            </div>
-            <div class="card">
-              <img src="images/card-img.PNG" />
-              <h1 class="course-information">
-                Learn Python: The Complete Python Programming Course
-              </h1>
-              <p>Avinash Jain, The Codex</p>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star"></span>
-              <span class="fa fa-star"></span>
-              <p>$94.99</p>
-            </div>
-            <div class="card">
-              <img src="images/card-img.PNG" />
-              <h1 class="course-information">
-                Learn Python: The Complete Python Programming Course
-              </h1>
-              <p>Avinash Jain, The Codex</p>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star"></span>
-              <span class="fa fa-star"></span>
-              <p>$94.99</p>
-            </div>
-            <div class="card">
-              <img src="images/card-img.PNG" />
-              <h1 class="course-information">
-                Learn Python: The Complete Python Programming Course
-              </h1>
-              <p>Avinash Jain, The Codex</p>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star"></span>
-              <span class="fa fa-star"></span>
-              <p>$94.99</p>
-            </div>
-            <div class="card">
-              <img src="images/card-img.PNG" />
-              <h1 class="course-information">
-                Learn Python: The Complete Python Programming Course
-              </h1>
-              <p>Avinash Jain, The Codex</p>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star checked"></span>
-              <span class="fa fa-star"></span>
-              <span class="fa fa-star"></span>
-              <p>$94.99</p>
-            </div>
-          </div>
+          <div class="course-card" id="cards"></div>
+          <script src="script.js">
+            getCourse("cards","");
+          </script>
+          <script>
+            getCourse("cards", "");
+          </script>

```

Comment:
if a script tag has `src` attribute there should be no content in the element.
```suggestion
          <script src="script.js"></script>
          <script>
            getCourse("cards", "");
          </script>

```

Reply:
Another tip for this is to avoid doing this here in a script tag at all, you can just put `getCourse("cards", "");` as the last line in your javascript file (outside any function) and it will be excuted on load just as you need.